versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/azure-marketplace-payg-integration/prerequisites.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Prerequisites
```

Comment:
Similar suggestions should be applied as for the AWS page

Reply:
Updated per a4e1f471c3673a920f4ac532c5fab41109ed9cbf